src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -64,7 +64,7 @@ azure-mgmt-netapp==8.0.0
 azure-mgmt-network==20.0.0
 azure-mgmt-policyinsights==1.1.0b2
 azure-mgmt-privatedns==1.0.0
-azure-mgmt-rdbms==10.0.0
+azure-mgmt-rdbms==10.1.0
```

Comment:
Considering the 10.2.0b1 is already released, double check whether the new SDK has the byok parameters included, and make the SDK version change either in this PR or in server log PR. No need to update it twice.

Reply:
Checked with version 10.2.0b1 and everything worked OK.